### DISCUSSION TITLE
Please help me review as soon as possible

### DISCUSSION THREAD

**Discussion by chenfan1227** [WalletConnect/walletconnect-monorepo#1828] [Developer Support]
Hi, we have submitted a modification request form, please review it
Account: info@cnuk.com
Wallet name: Hideout Wallet
thanks!!

**Comment by wongzigii** [comment]
https://github.com/WalletConnect/walletconnect-monorepo/discussions/1808
My Review is still pending after 3 weeks Please help
Wallet name: Loopring Wallet

**Comment by bkrem** [comment]
Hi @chenfan1227,

This seems to be the only relevant submission, which is in `Draft` state so we can't review. Please submit via the "Publish" button which should then show "Pending Review" as status afterwards:

<img width="794" alt="Screenshot 2023-01-05 at 16 13 15" src="https://user-images.githubusercontent.com/8663099/210813509-298294ff-40b9-4e95-854d-6345b7316340.png">

---

I searched for "Hideout Wallet", but alternatively please provide the value of `uuid=` in your URL when you're inspecting your submission so we can locate the correct submission if this isn't the right one.
